libr/core/cmd_anal.c

```diff
@@ -6143,7 +6133,7 @@ void _CbInRangeAav(RCore *core, ut64 from, ut64 to, int vsize, bool asterisk, in
 		r_cons_printf ("f+ aav.0x%08"PFMT64x "= 0x%08"PFMT64x, to, to);
 	} else {
 #if 1
-		r_anal_ref_add (core->anal, to, from, ' ');
+		r_anal_xrefs_set (core->anal, ' ', from, to);
```

Comment:
use the enum of R_ANAL_XREF_TYPE_ instead of ' '

Reply:
The problem is that there's no enum for ' '.
I don't know how that worked so far...